api/operator_license.go

```diff
@@ -0,0 +1,92 @@
+package api
+
+import (
+	"io/ioutil"
+	"strings"
+	"time"
+)
+
+type License struct {
+	// The unique identifier of the license
+	LicenseID string `json:"license_id" jsonapi:"primary,license"`
```

Comment:
What is the `jsonapi` struct tag for?

Reply:
This struct definition was mostly just ripped from our private licensing library + a couple extra Consul specific fields. The non consul specific bits were copy/pasted and the jsonapi struct tag was leftover from that. I have removed it.